src/Tribe/Admin/Display_Settings.php

```diff
@@ -13,7 +13,8 @@ class Tribe__Tickets__Admin__Display_Settings {
 	 * @since TBD
 	 */
 	public function hook() {
-		add_filter( 'tribe_display_settings_tab_fields', [ $this, 'add_display_settings' ] );
+		// @todo Uncomment this in G20.07
+		//add_filter( 'tribe_display_settings_tab_fields', [ $this, 'add_display_settings' ] );
```

Comment:
:no_entry_sign: No space found before comment text; expected "// add_filter( 'tribe_display_settings_tab_fields', [ $this, 'add_display_settings' ] );" but found "//add_filter( 'tribe_display_settings_tab_fields', [ $this, 'add_display_settings' ] );"

<sub>hash: 08e9be665bb78b41622876e3cf9f33cf</sub>